packages/fast-components-class-name-contracts-base/src/select/index.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * The class name contract for the select component
+ */
+export interface SelectClassNameContract {
+    /**
+     * The root of the select component
+     */
+    select?: string;
+
+    /**
+     * The disabled modifier
+     */
+    select__disabled?: string;
+
+    /**
+     * The menu of the select component
+     */
+    select_menu?: string;
+
+    /**
+     * The menu open modifier
+     */
+    select__menuOpen?: string;
```

Comment:
Since `select_menu` exists, in the sense of `block_element`, shouldn't this actually be `select_menu__open`? Same with below

Reply:
sure, done